src/app/[lang]/(future)/tools/metronome/page.tsx

```diff
@@ -1,18 +1,11 @@
 'use client';
 
-import { Add, Remove } from '@mui/icons-material';
-import {
-    Box,
-    Button,
-    IconButton,
-    Paper,
-    Slider,
-    TextField,
-    Typography,
-} from '@mui/material';
+import { Center, Column } from '@/components/shared/Layout';
+import { Paper } from '@mui/material';
 import { useEffect, useMemo, useState } from 'react';
-// import beatSound from './beat/beat.mp3';
-// import drumstickSound from './beat/drumsticks.mp3';
+import { Metre, MetronomeSlider, StopStartButton, StrongBeat, Tempo } from './_components';
+import beatSound from '../../../../../../public/sounds/beat.mp3';
```

Comment:
Maybe move the mp3 to the assets directory and import them with the shortcut "@assets"?

Reply:
Or you can create new import alias named "sounds" and then use @sounds to import them